common/remote_object.go

```diff
@@ -168,6 +171,14 @@ func parseRemoteObject(obj *cdpruntime.RemoteObject) (any, error) {
 		return parseRemoteObjectValue(obj.Type, obj.Subtype, string(obj.Value), obj.Preview)
 	}
 
+	if bigIntR.Match([]byte(obj.UnserializableValue)) {
```

Comment:
No strong opinion here, but, to make this slightly easier to read, I'd put `obj.UnserializableValue` into a variable, maybe named `uv`.

Reply:
Resolved in 99009e552ca56d8fd7050f3dc807eba4afe14369